sdks/java/container/java25/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+project.ext {
+    imageJavaVersion = '25'
```

Comment:
Could you open a jira like https://issues.apache.org/jira/browse/INFRA-27219?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel to get the java 25 dockerhub repo added?

Probably worth just asking for distroless as well even if we don't populate it for a while.